anomalib/utils/loggers/__init__.py

```diff
@@ -25,15 +26,49 @@
 from .tensorboard import AnomalibTensorBoardLogger
 from .wandb import AnomalibWandbLogger
 
-__all__ = ["AnomalibTensorBoardLogger", "get_logger", "AnomalibWandbLogger"]
+__all__ = [
+    "AnomalibTensorBoardLogger",
+    "AnomalibWandbLogger",
+    "get_console_logger",
+    "get_experiment_logger",
+]
 AVAILABLE_LOGGERS = ["tensorboard", "wandb", "csv"]
 
 
 class UnknownLogger(Exception):
     """This is raised when the logger option in `config.yaml` file is set incorrectly."""
 
 
-def get_logger(
+def get_console_logger(name: Optional[str] = None, level: Union[int, str] = logging.INFO) -> Logger:
```

Comment:
Maybe we could rename this `configure_logger` and instead of having it return the logger, just define `logger = logging.getLogger("anomalib")` in `train.py` and the other entrypoitns. That way the logging in those entrypoint files would be more in line with the other submodules.

Reply:
This would work if the existing `logger` were renamed to something like `experiment_logger` or similar

Reply:
Do you mean something like the following in `train.py` and other entrypoints?

```
...
configure_logger()
logger = logging.getLogger("anomalib")
...
```

If yes, this would cost us two lines for the sake of following the standard logging definition.

We could maybe rename this to `get_logger` instead, which would be a one liner in this case. And the experiment logger would also be renamed accordingly.
```
...
logger = get_logger("anomalib", level=logging.INFO)
...
experiment_logger = get_experiment_logger(config)
```

Reply:
> If yes, this would cost us two lines for the sake of following the standard logging definition.

It would add a single line `logger = logging.getLogger("anomalib")`, but this line would be placed at the top of the file, right under the imports. So it may be an additional line, but it does not clutter the `train` function. 

Also this same line would be removed from the `get_logger`/`configure_logger` function, which would end up a lot cleaner, because it would also only require a single argument (`log_level`), no logic for setting the logger name, and no return type.